src/installer/managed/Microsoft.NET.HostModel/ResourceUpdater.cs

```diff
@@ -352,128 +137,239 @@ public ResourceUpdater AddResource(byte[] data, string lpType, IntPtr lpName)
         /// </summary>
```

Comment:
This `Update` function depends on data in stream is same as `PEReader` created, so I think `Update` should throw `InvalidOperation` if Update() was already called instead of rewriting doc comments.